api/src/main/java/com/ford/labs/retroquest/email/EmailController.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2022 Ford Motor Company
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ford.labs.retroquest.email;
+
+import com.ford.labs.retroquest.exception.EmailNotAssociatedWithAnyTeamsException;
+import com.ford.labs.retroquest.team.Team;
+import com.ford.labs.retroquest.team.TeamService;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping(value = "/api/email")
+public class EmailController {
```

Comment:
The team controller is getting very cluttered and I feel like it makes more sense to have endpoints that are just sending emails be in their own controller. I'll move the other endpoints over soon.